autoscheduler/frontend/src/components/InfoPage/InfoPage.tsx

```diff
@@ -0,0 +1,94 @@
+out the course you want. Repeat this for every course you want to add to your schedule. During registration, 
+this information is updated about every 15 minutes.
+
+* If you do not want classes at a certain time, click and drag on the calendar
+to block off those &lsquo;&lsquo;busy times&rsquo;&rsquo; when generating schedules.
+To remove your busy times, you can click the trash icon. To quickly remove them in
+groups, you can click and hold on an edge then drag to rearrange multiple blocks so
+that all your times are on the same day before clicking the icon.
+
+* Hit the generate schedules button to generate up to 5 schedules with the options
+you selected. New schedules will overwrite all previous unsaved schedules.
+If you see a schedule you like, click the lock icon to save it.
+The icon should change so that the lock is now closed. You can also
+give the schedule a name to identify it easier by clicking the pencil icon.
+
+* You can click on a generated schedule to make it appear on the big calendar.
+You can also click the details button to open a pop up with more information
+about the sections in that schedule.
+
+* Whenever you click the generate button, only up to five possible schedules will be
+generated. This is because some courses have a lot of sections and showing every single
+possible combination at once can be be very overwhelming. If you are not satisfied
+with any of the generated schedules, hitting the button again will automatically
+remove all unsaved schedules and generate up to five more new ones.`;
+
+const advancedFilteringTitle = '## Advanced Filtering Options';
+
+const advancedFilteringList = `* You can choose to filter the sections included in schedule generation 
+at either the &lsquo;&lsquo;Basic&rsquo;&rsquo; or &lsquo;&lsquo;Section&rsquo;&rsquo; customization level. 
+When you click &lsquo;&lsquo;Generate Schedules&rsquo;&rsquo;, the currently selected customization 
+level for that course will be used.
+
+* Basic is recommended for when you are searching for sections with a specific attribute such as 
+&lsquo;&lsquo;honors&rsquo;&rsquo; or &lsquo;&lsquo;online&rsquo;&rsquo;. It displays available 
+attributes and allows you to decide whether you want to include sections that have those attributes 
+in schedule generation.
+
+* Section is recommended for when you already know which section numbers or professors you want. 
+It allows you to select the specific sections or professors you want to include in schedule generation.
+
+* At the top right of every course you add, there is a switch labeled 
+&lsquo;&lsquo;Include in schedules&rsquo;&rsquo;. When it is on the course name will have a maroon 
+background. When it is off it will have a gray background. Only courses with this switch set to on 
+will be included in your schedules when you hit generate. This feature is useful when you want to 
+quickly generate schedules with different combinations of courses.`;
```

Comment:
This should definitely be included, but I feel like we need to organize this part so that if you want to look up stuff about "sort by" or "include in schedules" you can easily find info about them under here. Maybe we could bold the words and put them at the start of the bullets like:

### Advanced Filtering Options

- **Include in schedules**: Stuff
- **Sort by**: stuff

Alternatively we could give them their own sections (with four `#`s) but I think that's probably overkill, e.g.

### Advanced Filtering Options
#### Include in schedules
stuff
#### Sort by
stuff